### PR TITLE
Git Workshop 2023 Assignment

### PR DESCRIPTION
**Pull request by rpandox**

### CODE REVIEW

File: 2023/assignments/Roshan/Roshan.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: remove redundant space at last

**Comment by SusheelThapa** [comment]
image

Upto 9 number timile lekeko format right xa,
Ani 10 pachi tyo Description ma **Description** ako xan, tyo update gara...

**Comment by rpandox** [comment]
Resolved all the problems 
Can you check them again 
```
@SusheelThapa 
Thankyou for the feedback
```

File: 2023/assignments/Roshan/Roshan.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    It is used to point to an existing repo and make a clone or copy of that repo at in a new directory, at another location.
+
+**Examples**:
+
+        $ git clone https://github.com/rpandox/First-Contribution.git
+
+15.  `git branch`
+
+***Description** :
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: One asterisk is more...

File: 2023/assignments/Roshan/Roshan.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>`Roshan Pandey`</p>
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: Remove back tick(**\`**) from the name.

File: 2023/assignments/Roshan/Roshan.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+17.  `git rebase`
+
+**Description** :
+
+        It is used to changing the base of your branch from one commit to another making it appear as if you'd created your branch from a different commit
+
+**Examples**:
+
+        $ git rebase branch_name
+
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: mistake

**Review by SusheelThapa — CHANGES_REQUESTED** [review]

**Comment by rpandox** [comment]
Can you please check again I updated the file 
@SusheelThapa

File: 2023/assignments/Roshan/Roshan.md (hunk 3 above)
Comment (SusheelThapa) [reply 2023/assignments/Roshan/Roshan.md]: You haven't resolve this change.

File: 2023/assignments/Roshan/Roshan.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: Indent alignment mistake xa hai 17 ma
@rpandox

**Review by SusheelThapa — CHANGES_REQUESTED** [review]

**Comment by SusheelThapa** [comment]
Ani tyo name ma backtick wala ne solve gara @rpandox 
image

File: 2023/assignments/Roshan/Roshan.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/github.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+       <a href="https://www.instagram.com/roshan_pandey097/" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/instagram.svg"
+        alt="your_name"
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: In the *alt='your_name'*, replace with your name.

File: 2023/assignments/Roshan/Roshan.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+    <a href="https://github.com/rpandox" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/github.svg"
+        alt="your_name"
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: In the *alt='your_name'*, replace with your name.

File: 2023/assignments/Roshan/Roshan.md [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/facebook.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+    <a href="https://www.linkedin.com/in/rpandox/" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg"
+        alt="your_name"
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: In the *alt='your_name'*, replace with your name.

File: 2023/assignments/Roshan/Roshan.md [hunk 9]
```diff
@@ -0,0 +1,232 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>`Roshan Pandey`</p>
+
+<p align="center">
+    <a href="https://www.facebook.com/R.Pandey198/" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/facebook.svg"
+        alt="your_name"
```
Comment (SusheelThapa) [inline 2023/assignments/Roshan/Roshan.md]: In the *alt='your_name'*, replace with your name.

**Review by SusheelThapa — CHANGES_REQUESTED** [review]

**Comment by rpandox** [comment]
I think this is the final change 
@SusheelThapa

**Review by SusheelThapa — APPROVED** [review]

**Pull request merged**